app/Notifications/PasswordReset.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Notifications;
+
+use Carbon\Carbon;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+use Illuminate\Support\Facades\Lang;
+
+/**
+ * This class provides the notification for password reset emails.
+ *
+ * @package App\Notifications
+ */
+class PasswordReset extends Notification
+{
+    use Queueable;
+
+    /**
+     * The password reset token.
+     *
+     * @var string
+     */
+    private $token;
+
+    /**
+     * The date when the password will expire.
+     *
+     * @var Carbon
+     */
+    private $expireDate;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param string $token
+     * @param Carbon $expireDate
+     */
+    public function __construct(string $token, Carbon $expireDate)
```

Comment:
the parameter expireDate is a bit missleading, as it is just the start date to which the expire time is added to. I'm not even sure if we need it at all.

Reply:
Why we shouldn't need this date? What about the expire time in the email?

Reply:
As the expireDate in the constructor = now, you can get remove the parameter and set the expire data = now + expire_minutes... in the toMail method. That way you also don't steal time from the user if the email sending process get's delayed.

Reply:
I'm sorry! I misunderstood something.